docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -168,4 +168,6 @@ Name                                                         Description
                                                              that has been configured for the Trino coordinator.
```

Comment:
commit message too long

Reply:
the toggle name is intentionally long and it would be useless to omit it